.github/workflows/ci.yml

```diff
@@ -21,6 +17,19 @@ jobs:
           export NIX_PATH=nixpkgs=$(jq '.nixpkgs.url' nix/sources.json -r)
           echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
           nix-shell shell.nix --run "echo"
+      - name: Go checks
+        run: |
+          nix-shell shell.nix --run "./scripts/go-checks.sh"
       - name: BootStrap k8s cluster
         run: |
-          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --mayastor --zfs --lvm"
+          nix-shell shell.nix --run "./scripts/k8s/deployer.sh start --workers 1 --zfs --lvm" 
+      - name: E2E smoke test
+        run: |
+          nix-shell shell.nix --run "./scripts/e2e-test.sh --testplan selfci"
+      - name: Setup tmate session
+        if: ${{ failure() }}
+        timeout-minutes: 15
+        uses: mxschmitt/action-tmate@v3
+        with:
+          detached: true
```

Comment:
Is detach needed in this case?

Reply:
It's not required here because there is no steps in workflow after this tmate session steps.

```
In detached mode, this Action will start the tmate session, print the connection details, and continue with the next step(s) of the workflow's job.
```

Reply:
[fixed](https://github.com/openebs/openebs-e2e/compare/aac2d9ae2cec3b80cd89fee4cb1ff57ddc44c215..61f53a539d78cb3e47dd205a316b7c0abc9840b0)